ansible.cfg

```diff
@@ -3,13 +3,15 @@
 
 # This config file provides examples for running
 # the OpenShift playbooks with the provided
-# inventory scripts. Only global defaults are
-# left uncommented
+# inventory scripts.
 
 [defaults]
 # Add the roles directory to the roles path
 roles_path = roles/
 
+# Add the callback_plugins path
+callback_plugins = callback_plugins/
+
```

Comment:
Should we also add some sane defaults? I would suggest the following at minimum:
default:
- forks=20
- host_key_checking=False
- retry_files_enabled=False
- no_cows=True
- log_path=/tmp/ansible.log (commented out)

ssh_connection:
- pipelining=True